source/browseMode.py

```diff
@@ -311,6 +311,15 @@ def _quickNavScript(self,gesture, itemType, direction, errorMessage, readUnit):
 
 	@classmethod
 	def addQuickNav(cls, itemType, key, nextDoc, nextError, prevDoc, prevError, readUnit=None):
+		"""Adds a script for the given quick nav item.
+		@param itemType: The type of item, I.E. "heading" "Link" ...
+		@param key: The quick navigation key to bind to the script. Shift is automatically added for the previous item gesture.
+		@param nextDoc: The input help mode string to bind to the script that yields the next quick nav item.

```

Comment:
Change "input help mode string" to "command description" or, if you don't think people will get this, "input help mode/input gestures dialog description".